dvc/utils/__init__.py

```diff
@@ -336,7 +336,7 @@ def env2bool(var, undefined=False):
 def resolve_output(inp, out):
     from urllib.parse import urlparse
 
-    name = os.path.basename(urlparse(inp).path)
+    name = os.path.basename(os.path.normpath(urlparse(inp).path))
```

Comment:
:thinking: hard to know that it was originally wrong (before `normpath`). Wondering if there's a better way to handle this cases without thinking too much about _correctness_.

Reply:
good catch, @pared :+1: